.github/ISSUE_TEMPLATE/QA_env_deployment.md

```diff
@@ -0,0 +1,18 @@
+# Description 
+<!-- Describe the particularity of this env if necessary -->
+
+# Test acceptance (Mandatory)
+
+Infra team:
+- [] How to connect is create
+- [] Presence of necessary passwords in bitwarden
+- [] Check if at least 2 people can work at the same time
+
```

Comment:
- verify we can connect to the main web components from the Bastion or the dev workstation:
  - admin
  - dagit
  - cloudbeaver
  - keycloak (admin)
- verify the metrics are pushed in AZmonitoring
- verify the TLS is configured (no auto-signed certificate)